code/modules/uplink/uplink_item.dm

```diff
@@ -110,10 +110,10 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	var/obj/item/weapon/storage/box/B = A
 	if(istype(B) && B.contents.len > 0)
 		for(var/obj/item/I in B)
-			U.purchase_log += "<big>[bicon(I)]</big>"
+			U.purchase_log += "<big>[icon2base64html(I)]</big>"
 	else
 		if(purchase_log_vis)
-			U.purchase_log += "<big>[bicon(A)]</big>"
+			U.purchase_log += "<big>[icon2base64html(A)]</big>"
```

Comment:
the other solution would be store the icons/items to send in a more structured way, and generate the html part at round end.